src/pages/VenuePage/VenuePage.tsx

```diff
@@ -224,21 +351,41 @@ export const VenuePage: React.FC = () => {
 
     if (isEventStartingSoon(event)) {
       return (
-        <CountDown
-          startUtcSeconds={event.start_utc_seconds}
-          textBeforeCountdown="Bar opens in"
-        />
+        <>
+          <Preload venue={venue} />
+          <Suspense fallback={<></>}>
+            <CountDown
+              startUtcSeconds={event.start_utc_seconds}
+              textBeforeCountdown="Bar opens in"
+            />
+          </Suspense>
+        </>
       );
     }
   }
 
-  if (!user || isRelatedVenuesLoading) {
-    return <LoadingPage />;
+  // @debt there is already !user check above, this is superfluous
+  if (!user) {
+    return (
+      <>
+        <Preload venue={venue} />
+        <Suspense fallback={<></>}>
+          <LoadingPage />
+        </Suspense>
+      </>
+    );
   }
 
   if (profile && !isCompleteProfile(profile)) {
     history.push(`/account/profile?venueId=${venueId}`);
   }
 
-  return <TemplateWrapper venue={venue} />;
+  return (
+    <>
+      <Preload venue={venue} />
+      <Suspense fallback={<></>}>
+        <TemplateWrapper venue={venue} />
+      </Suspense>
+    </>
+  );
```

Comment:
As per previous comment, this should probably be reverted due to conflicts with:

- https://github.com/sparkletown/sparkle/pull/1737

Reply:
Preload is now done with a hook for `VenuePage` and removing the lazy load from `TemplateWrapper` showed increase of _Cumulative Layout Shift_ metric from the range of **0.01** - **0.02** to the higher order of magnitude **0.1** .

I'm assuming this is because eager `TemplateWrapper`  shows up too early and resizes  after the map background gets fully loaded.

Electing to keep it lazy in here.

Reply:
Other lazy loaded components (especially in `TemplateWrapper`) are with b7ec26dc13ee3c39f4d061fe3565ff7cd02b0f3e in line with staging now.

Reply:
Can you please show the explicit stats on what having this lazy loaded or not does for our bundle metrics as per the screenshots/etc I provided in my 'split' PRs.

I'm still not convinced that this is going to be of any major benefit, and IMO the fact that the CLS changes is more of a side effect of a race condition for a problem we can fix better in future by explicitly setting the appropriate size on the map image/container ahead of time.

Reply:
> IMO the fact that the CLS changes is more of a side effect of a race condition for a problem we can fix better in future by explicitly setting the appropriate size on the map image/container ahead of time.

Maybe side effect of lazy loading, perceived load is still improved with it. Should there be PR in the future that improves upon it (don't know if I'd call async loading of assets race condition due to connotation) that PR can revert the lazy loading of components as well.

This PR is about load performance of `VenuePage` so if it is lazy or not, once it gets loaded, that performance is of scope here.

Reply:
Here is the table I've compiled
![sparkle-preload-assets-01](https://user-images.githubusercontent.com/79229621/125069086-9269a980-e0b6-11eb-889c-72a5246d918f.png)

Reply:
My point here is about not adding arbitrary bundle splits without proving their relevance and showing the impact they have..